src/community/template-editor/src/main/resources/GeoServerApplication.properties

```diff
@@ -0,0 +1,31 @@
+TemplateEditorPage.page.title=Hello
```

Comment:
Some of these probably need a little more work or maybe can be removed?

Reply:
woops. Can probably be removed. I'll check. Thanks for spotting this out